.github/workflows/unmanaged_dependency_check.yaml

```diff
@@ -12,15 +12,8 @@ jobs:
         java-version: 11
     - name: Install modules
       shell: bash
-      run: |
-        mvn -B -V -ntp -DskipTests=true \
-        -Dclirr.skip=true \
-        -Denforcer.skip=true \
-        -Dmaven.javadoc.skip=true \
-        -Dgcloud.download.skip=true \
-        -T 1C \
-        clean install
+      run: .kokoro/build.sh
     - name: Unmanaged dependency check
-      uses: googleapis/sdk-platform-java/java-shared-dependencies/unmanaged-dependency-check@unmanaged-dependencies-check-latest
+      uses: googleapis/sdk-platform-java/java-shared-dependencies/unmanaged-dependency-check@google-cloud-shared-dependencies/v3.24.0
```

Comment:
Can we try 3.23.0 (previous version) without failing the build? If you use 3.23.0 and RenovateBot config is correct, you'll get a pull request soon (today) rather than waiting for next sdk-platform-java release.

Reply:
Sure, will do.